templates/repo/actions/workflow_dispatch.tmpl

```diff
@@ -0,0 +1,72 @@
+<div class="ui ignored info message project-column-header" >
+	<span class="ui left floated text middle">{{ctx.Locale.Tr "actions.workflow.has_workflow_dispatch"}}</span>
+	<button class="ui right floated mini button show-modal" data-modal="#runWorkflowDispatchModal">{{ctx.Locale.Tr "actions.workflow.run"}}{{svg "octicon-triangle-down" 14 "dropdown icon"}}</button>
+</div>
+<div id="runWorkflowDispatchModal" class="ui mini modal">
+	<div class="content">
+		<form id="runWorkflowDispatchForm" class="ui form" action="{{$.Link}}/run?workflow={{$.CurWorkflow}}&actor={{$.CurActor}}&status={{.Status}}" method="post">
+			{{.CsrfTokenHtml}}
+			<div class="ui inline field required">
+				<label>{{ctx.Locale.Tr "actions.workflow.from_ref"}} :</label>
+				<div class="ui floating dropdown icon button mini">
+					<input type="hidden" name="ref" value="refs/heads/{{index .Branches 0}}">
+					<i>{{svg "octicon-git-branch"}}</i>
+					<span class="text" style="font-size: medium">{{index .Branches 0}}</span>
+					<div class="menu">
+						<div class="ui icon search input">
+							<i class="search icon">{{svg "octicon-search" 16 "search icon"}}</i>
+							<input type="text" placeholder="{{ctx.Locale.Tr "repo.filter_branch_and_tag"}}">
+						</div>
+						<div class="scrolling menu">
+							<div class="header">
+								{{svg "octicon-git-branch"}}
+								Branches
+							</div>
+							{{range $key, $item := .Branches}}
+							<div class="item {{if eq $key 0}}active selected{{end}}" data-value="refs/heads/{{.}}">{{.}}</div>
+							{{end}}
+							<div class="divider"></div>
+							<div class="header">
+								{{svg "octicon-tag"}}
+								Tags
+							</div>
+							{{range $key, $item := .Tags}}
+							<div class="item" data-value="refs/tags/{{.}}">{{.}}</div>
+							{{end}}
+						</div>
+					</div>
+				</div>
+			</div>
+
+			<div class="divider"></div>
+			{{range $key, $item := .WorkflowDispatchConfig.Inputs}}
+			<div class="ui field {{if .Required}}required{{end}}">
+				{{if eq .Type "choice"}}
+					<label>{{$key}} :</label>
+					<select class="ui fluid dropdown" name="{{$key}}">
+						{{range .Options}}
+						<option value="{{.}}" {{if eq $item.Default .}}selected{{end}} >{{.}}</option>
+						{{end}}
+					</select>
+					<span class="help">{{.Description}}</span>
+				{{else if eq .Type "boolean"}}
+					<div class="ui inline toggle checkbox">
+						<label>{{$key}}</label>
```

Comment:
No `:`?

Reply:
Yes, the label is behind the checkbox

![image](https://github.com/go-gitea/gitea/assets/3114995/65559d5a-64d8-4dc8-b429-d7f220ebeae7)